### PR TITLE
new readme

### PR DESCRIPTION
**Pull request by rachelzhang0922**
**What this PR does / why we need it**:

**Will this PR make the community happier**? 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**How is this PR tested**
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Review by saiyan86 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -18,46 +18,49 @@
 
 </div>
 
-***Shifu*** is a [Kubernetes](https://kubernetes.io) native framework designed to abstract out the complexity of interacting with IoT devices. ***Shifu*** aims to achieve TRUE plug-and-play IoT device virtualization, management, control, and automation.
+***Shifu*** is the next-generation open source IoT development framework, making developing an IoT app as simple as developing a Web app. The high extensibility of ***Shifu*** allows it to integrate almost all devices and protocols. Once integrating a device, ***Shifu*** generates a digital twin of the device as a microservice in the system and opens up device data and capabilities in the form of APIs. In this way, traditional IoT application development is reduced to simple web development, which greatly improves the efficiency, quality and reusability of IoT application development.
```
Comment (BtXin) [inline README.md:21]: `making developing an IoT app as simple as developing a Web app` - > `makes developing an IoT app as simple as developing a Web app`
`Once integrating a device` -> `Once a device is integrated`

File: README.md [hunk 2]
```diff
@@ -18,46 +18,49 @@
...
-Developers and operators should focus entirely on using their creativity to make huge impacts, rather than fixing infrastructure and reinventing the wheel on a daily basis. As a developer and an operator, ***Shifu*** knows your pain!!! ***Shifu*** wants to take out all the underlying mess, and make us developers and operators happy again!
+For more information please visit our [documentation](https://shifu.run/docs/).
 
-## Software-Defined World
+## Design principle
 
-If every IoT device has a ***deviceShifu*** with it, we can leverage software to manage our surroundings, and make the whole world software defined. In a software-defined world, everything is intelligent, and the world around you will automatically change itself to serve your needs -- after all, all the technology we built is designed to serve human beings.
+Our job is to make developers and operators happy. Which is why all our designs are developer-oriented:
+#### 📡Easy to deploy
+Shifu can always be deployed with a single command.
```
Comment (BtXin) [inline README.md:57]: ~~always~~

File: README.md [hunk 3]
```diff
@@ -18,46 +18,49 @@
...
+#### 🤖Plug'n'Play
+Shifu should automatically recognize and provide basic functionalities to a supported IoT device. Once the developer completes Shifu's template, all features of the device should be immediately available.
```
Comment (BtXin) [inline README.md:59]: `Shifu should automatically recognize and provide basic functionalities to a supported IoT device` -> `Shifu will automatically recognize and provide basic functionalities to a supported IoT device`

File: README.md [hunk 4]
```diff
@@ -18,46 +18,49 @@
...
+#### 🪄Easy to extend
+Developer can further implement Shifu's interface/SDKs to create custom features and unleash endless possibilities.
```
Comment (BtXin) [inline README.md:61]: ~~SDKS~~
we don't have sdks right now.

File: README-zh.md [hunk 5]
```diff
@@ -18,46 +18,49 @@
...
-## 为什么用 ***Shifu***
 
-***Shifu*** 通过数字孪生技术，让管理和控制IoT设备变得极其简单。当您连接设备的时候，***Shifu*** 会识别并以一个 [Kubernetes Pod](https://kubernetes.io/zh-cn/docs/concepts/workloads/pods/) 的方式启动一个该设备的数字孪生 ***deviceShifu***。
+|特点<div style="width: 80pt">|  |
+|---|---|
+|💻极速|大到使用私有驱动的工程机械，小到使用公有协议的温湿度计，Shifu的超高兼容性能让你轻松应对各种异构设备。|
+|▶️模块化|所有接入Shifu的设备及应用都会被封装成一个个拼图式模块，根据场景内的设备不同, 按需加载即可。|
+|👨‍💻高效|接入设备后，Shifu会自动把设备的能力抽象成API，让你的应用和硬件设备彻底解耦，让低效的物联网应用开发变得像面向对象编程一样高效。|
+|🚀稳定  |Shifu已通过在航天场景验证，提供99.9999%的可靠性, 让你远离宕机烦恼。|
+|🛡️安全|联合国的云原生安全团队操刀，无论是数据加密还是网络安全，Shifu不一而足。|
```
[the remark this answers is not in the document]
Comment (saiyan86) [reply README-zh.md:30]: |🛡️安全|联合国的云原生安全团队操刀，无论是数据加密还是网络安全，Shifu均可无缝集成。|

File: README-zh.md [hunk 6]
```diff
@@ -18,46 +18,49 @@
...
-### 软件定义世界
-
-如果每一个IoT设备都有一个 ***deviceShifu***，我们就可以借助软件来管理我们周围的世界。在一个软件定义的世界 (Software Define World) 中，所有东西都是智能的。您周围的一切会自动改变，进而更好地服务您；因为归根到底，科技以人为本。
+## 设计理念
+#### 📡 易于部署
+Shifu必须只用一个命令即可完成部署。
+#### 🤖即插即用
+Shifu必须自动识别并为一个新的物联网设备提供基本功能。一旦开发者完成了Shifu的模板，设备的所有功能就应该立即可用。
+#### 🪄易于扩展
+开发者可以进一步实现Shifu的接口/SDK，以创建自定义功能，释放出无限的可能性。
```
[the remark this answers is not in the document]
Comment (saiyan86) [reply README-zh.md:61]: 马上就有了，我有一个PR在加shifuctl

[the remark this answers is not in the document]
Comment (BtXin) [reply README-zh.md:61]: > 马上就有了，我有一个PR在加shifuctl

ok

File: README-zh.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (BtXin) [reply README-zh.md:30]: 我在想是`前联合国的云原生安全团队`还是`联合国的前云原生安全团队`哪个比较好

[the remark this answers is not in the document]
Comment (rachelzhang0922) [reply README-zh.md:30]: 联合国前团队，不然语义有问题了

File: README.md [hunk 7]
```diff
@@ -18,46 +18,49 @@
 
 </div>
 
-***Shifu*** is a [Kubernetes](https://kubernetes.io) native framework designed to abstract out the complexity of interacting with IoT devices. ***Shifu*** aims to achieve TRUE plug-and-play IoT device virtualization, management, control, and automation.
+***Shifu*** is the next-generation open source IoT development framework, makes developing an IoT app as simple as developing a Web app. The high extensibility of ***Shifu*** allows it to integrate almost all devices and protocols. Once a device is integrated, ***Shifu*** generates a digital twin of the device as a microservice in the system and opens up device data and capabilities in the form of APIs. In this way, traditional IoT application development is reduced to simple web development, which greatly improves the efficiency, quality and reusability of IoT application development.
```
Comment (tomqin93) [inline README.md:21]: reduced to => simplified to

File: README.md [hunk 8]
```diff
@@ -18,46 +18,49 @@
...
 
-## Why ***Shifu***
+|Features<div style="width: 100pt">|  |
+|---|---|
+|💻 Ultra-fast|From thermohydrometers using standard protocols to complex machinery using proprietary drivers, Shifu is capable of integrating all kinds of heterogeneous devices.|
```
Comment (tomqin93) [inline README.md:25]: Blazingly fast

File: README.md [hunk 9]
```diff
@@ -18,46 +18,49 @@
...
-## Software-Defined World
+## Design principle
 
-If every IoT device has a ***deviceShifu*** with it, we can leverage software to manage our surroundings, and make the whole world software defined. In a software-defined world, everything is intelligent, and the world around you will automatically change itself to serve your needs -- after all, all the technology we built is designed to serve human beings.
+Our job is to make developers and operators happy. Which is why all our designs are developer-oriented:
+#### 📡Easy to deploy
+Shifu can be deployed with a single command.
+#### 🤖Plug'n'Play
+Shifu will automatically recognize and provide basic functionalities to a supported IoT device. Once the developer completes Shifu's template, all features of the device should be immediately available.
+#### 🪄Easy to extend
```
Comment (tomqin93) [inline README.md:60]: Extensibility ?

File: README.md [hunk 10]
```diff
@@ -18,46 +18,49 @@
...
+Developer can further implement Shifu's interface/SDKs to create custom features and unleash endless possibilities.
+#### 🔧Zero maintenance
+Shifu aims to achieve zero maintenance by adopting cutting-edge cloud native best practices. After all, Shifu should take care of himself and make operators' lives easier!
```
Comment (tomqin93) [inline README.md:63]: opeartors => DevOps

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@rachelzhang0922 added some comment, please take a look

File: README.md [hunk 11]
```diff
@@ -18,46 +18,49 @@
...
 
-## Why ***Shifu***
+|Features<div style="width: 100pt">|  |
+|---|---|
+|💻 Blazing-fast|From thermohydrometers using standard protocols to complex machinery using proprietary drivers, Shifu is capable of integrating all kinds of heterogeneous devices.|
+|▶️ Modularized|All devices and Apps integrated into Shifu are packaged as modules that can be loaded on demand.|
+|👨‍💻 Efficient|Once a device is integrated, Shifu automatically abstracts its capabilities into APIs, completely decoupling your App from the hardware, making IoT App development simple and efficient.|
+|🚀 Stable|Shifu is running in multiple production scenarios with 99.9999% stability, relieving you from the operational mess.|
+|🛡️ Safe |Designed by ex-UN cloud native security team. Shifu can easily enforce data encryption, network security and much more.|
+|🌐 Global Community|Benefiting from its Kubernetes' native architecture, Shifu can seamless integrate with the powerful cloud native software eco-system that allows developers around the world to help you with your problems.|
```
Comment (tomqin93) [inline README.md:30]: Let's be consistent with the Emojis for both versions

File: README.md [hunk 12]
```diff
@@ -18,46 +18,49 @@
 
 </div>
 
-***Shifu*** is a [Kubernetes](https://kubernetes.io) native framework designed to abstract out the complexity of interacting with IoT devices. ***Shifu*** aims to achieve TRUE plug-and-play IoT device virtualization, management, control, and automation.
+***Shifu*** is the next-generation open source IoT development framework, makes developing an IoT app as simple as developing a Web app. The high extensibility of ***Shifu*** allows it to integrate almost all devices and protocols. Once a device is integrated, ***Shifu*** generates a digital twin of the device as a microservice in the system and opens up device data and capabilities in the form of APIs. In this way, traditional IoT application development is simplified to simple web development, which greatly improves the efficiency, quality and reusability of IoT application development.
```
Comment (tomqin93) [inline README.md:21]: simplified to simple web development => simplified to web development

**Review by tomqin93 — COMMENTED** [review]
@rachelzhang0922 added 2 comment, the rest looks good

**Review by tomqin93 — APPROVED** [review]
LGTM

**Review by saiyan86 — APPROVED** [review]

**Comment by saiyan86** [comment]
LGTM

**Pull request merged**